src/chains/ethereum/options/src/logging-options.ts

```diff
@@ -87,28 +100,53 @@ export const LoggingOptions: Definitions<LoggingConfig> = {
   },
   quiet: {
     normalize,
-    cliDescription: "Set to `true` to disable logging.",
+    cliDescription: "Set to `true` to disable writing logs to the terminal.",
```

Comment:
Maybe something like:

```suggestion
    cliDescription: "Set to `true` to disable writing logs to `logger.log` (`stdout` by default).",
```